src/index.js

```diff
@@ -16,7 +19,37 @@ const server = new ApolloServer({
   instrospection: true,
   playground: true,
   tracing: true,
-  context: { models }
+  context: async ({ req, res }) => ({ models, user: req.user })
+})
+
+var opts = {}
+opts.jwtFromRequest = ExtractJwt.fromAuthHeaderAsBearerToken()
+opts.secretOrKey = 'secret'
+
+passport.use(
+  new JwtStrategy(opts, async function (jwt_payload, done) {
+    try {
+      const userFound = await models.User.findOne({ where: { id: jwt_payload.id } })
+
+      if (!userFound) {
+        return done(null, false)
+      }
+
+      done(null, userFound)
+    } catch (error) {
+      done(error, false)
+    }
```

Comment:
There is no need of an IF statement if you are already returning what ever the `models.User.findOne` result is, so let's change shrink this code a bit:

```suggestion
    try {
      done(null, await models.User.findOne({ where: { id: jwt_payload.id } }))
    } catch (error) {
      done(error)
    }
```